modules/rostests/winetests/user32/input.c

```diff
@@ -79,8 +79,10 @@ static struct {
     BOOL sendinput_broken;
 } key_status;
 
+#if (_WIN32_WINNT >= 0x0602)
```

Comment:
Make diff more explicit?
```suggestion
#if !defined(__REACTOS__) || _WIN32_WINNT >= 0x0602
```

Reply:
And use `_WIN32_WINNT_WIN8`.